lib/oban/crontab/cron.ex

```diff
@@ -40,6 +40,13 @@ defmodule Oban.Crontab.Cron do
   defp matches_rule?(:weekdays, weekday, datetime), do: weekday == Date.day_of_week(datetime)
 
   @spec parse!(input :: binary()) :: t()
```

Comment:
It would be great adding some documentation while you're here, including the standard and non-standard crontab expressions.